diff -ru expat-2.1.0/conftools/config.sub expat-2.1.0-sub/conftools/config.sub
--- expat-2.1.0/conftools/config.sub	2012-03-24 19:12:39.000000000 +0000
+++ expat-2.1.0-sub/conftools/config.sub	2013-12-29 12:50:08.000000000 +0000
@@ -247,6 +247,7 @@
 	# Some are omitted here because they have special meanings below.
 	1750a | 580 \
 	| a29k \
+        | aarch64 | aarch64_be \
 	| alpha | alphaev[4-8] | alphaev56 | alphaev6[78] | alphapca5[67] \
 	| alpha64 | alpha64ev[4-8] | alpha64ev56 | alpha64ev6[78] | alpha64pca5[67] \
 	| am33_2.0 \
@@ -352,6 +353,7 @@
 	# Recognize the basic CPU types with company name.
 	580-* \
 	| a29k-* \
+        | aarch64-* | aarch64_be-* \
 	| alpha-* | alphaev[4-8]-* | alphaev56-* | alphaev6[78]-* \
 	| alpha64-* | alpha64ev[4-8]-* | alpha64ev56-* | alpha64ev6[78]-* \
 	| alphapca5[67]-* | alpha64pca5[67]-* | arc-* \
Only in expat-2.1.0-sub/conftools: config.sub.orig
